bootstrap.py

```diff
@@ -639,6 +639,26 @@ def get_api_port():
 print "This script is designed to register new systems or to migrate an existing system to a Foreman server with Katello"
 
 
+def check_prerequisite():
+    rpm_sat = "katello-capsule"
```

Comment:
Sure do. Upstream you have `katello` and `katello-capsule` (pre 3.3) (/me forgot that Katello 3.2 and lower used `katello-capsule`. Katello > 3.3 you'd want to check for `foreman-proxy-content`. Downstream you'd want to check for `satellite`and `satellite-capsule`.  Though in fairness, if you run `bootstrap.py` on a system functioning as a Smart Proxy with content or a Capsule, it is already registered and `bootstrap.py` will fail because the system already has the CA certificate. But it doesn't hurt to be much more explicit.